docs/iiif_server_settings.md

```diff
@@ -0,0 +1,85 @@
+---
+title: IIF Server Settings Form Default Settings
+tags:
+  - IIIF
+  - IIIF Server Settings Form
+  - IIIF Content Search API
+  - Solr
+  - Solr Fields
+  - Solr Index
+---
+
+# IIIF Server Settings Form Default Settings
+
+The IIIF Server Settings Form is used to configure different IIIF related settings used throughout your Archipelago environment. We strongly advise keeping the default settings intact. The necessary [Solr Fields](strawberry_key_name_providers.md#creating-a-solr-field) listed below should be setup by default.
+
+You can find the IIIF Configuration Form:
+
+- Through the `Manage` menu > `Configuration` > `Archipelago` > `Configure Strawberry Runners Post Processors`
+- Directly at `/admin/config/archipelago/iiif` 
+
+![IIIF Server Settings Form](images/iiif_server_settings_form.png)
+
+On the IIIF Server Settings Form page, you will see the following:
+
+
+1. Note that these 'IIIF Server configuration URLs are used as defaults for field formatters using IIIF, but can be overridden on a one by one basis when setting up your formatters for each Display Mode.'
+    
+2. Base URL of your IIIF Media Server public accessible from the Outside World.
+    - Please provide a publicly accessible IIIF server URL. This URL will be used for AJAX and JS calls. Trailing Slashes will be removed.
+    - Set to `http://localhost:8183/iiif/2` by default.
+    - We do not recommend changing this selection. 
+
+3. Base URL of your IIIF Media Server accessible from inside this Webserver.
+    - Please provide Internal IIIF server URL. This URL will be used by Internal Server calls and needs to be locally accessible by your server, e.g 127.0.0.1 or an local Docker alias. Trailing Slashes will be removed.
+    - Set to `http://esmero-cantaloupe:8182/iiif/2` by default.
+    - We do not recommend changing this selection.
+
+4. Checkbox to 'Enable IIIF Content Search API V1 and V2 endpoints'.
+    - Checked by default in later (1.4.0+) versions of Archipelago.
```

Comment:
Same as with my other comment, we can point to a snippet showing how this is used in a IIIF Manifest? We will ship 1.4.0 with updated templates too

Reply:
Hmm, a snippet of what exactly @DiegoPino are you thinking? The section below has the full paths noted...

Reply:
A twig template snippet/piece ... ?
```TWIG
{%- if attribute(data, 'ispartof') %} 
  "within": "{{ parentnodeurl }}", 
  "partOf": [ { "id": "{{ parentnodeurl }}/metadata/iiifmanifest3cws/default.jsonld", "type": "Manifest" } ], 
{%- endif -%}
{# Allison: add note why we have 3 services. SearchService2 is the current one, no viewer supports it, SearchService1 is for Mirador, http://iiif.io/api/search/0/search is for UV #}
{%- if node.uuid.value -%}
    "service": [ 
        { 
            "id": "{{ baseurl }}iiifcontentsearch/v2/do/{{ node.uuid.value }}/metadatadisplayexposed/iiifmanifest/mode/advanced/page/0", 
            "type": "SearchService2" 
        }, 
        { 
            "id": "{{ baseurl }}iiifcontentsearch/v1/do/{{ node.uuid.value }}/metadatadisplayexposed/iiifmanifest/mode/advanced/page/0", 
            "type": "SearchService1", 
            "@context": "http://iiif.io/api/search/1/context.json", 
            "profile": "http://iiif.io/api/search/1/search" 
        }, 
        { 
            "@id": "{{ baseurl }}iiifcontentsearch/v1/do/{{ node.uuid.value }}/metadatadisplayexposed/iiifmanifest/mode/advanced/page/0", 
            "@context": "http://iiif.io/api/search/0/context.json", 
            "profile": "http://iiif.io/api/search/0/search" 
        } 
    ], 
{%- endif -%}
"items": [ 
```

Reply:
Where `/metadatadisplayexposed/iiifmanifest/` needs to match the actual exposed template that is using the current template being edited/enriched with this snippet .. could be another *like if your backend e.g provides extra data to be searched you are not showing to the user ... kinda cool feature I added to hide data from harvesters)

Reply:
Ok, that is the snippet I already added to the IIIF Content Search API overview, wonder if it's helpful to just refer people back to that doc and the section?